pkg/asset/tls/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
`*.bazel` no bazel files required. can you remove this.

Reply:
Yep, this got left in accidentally in one of the merges. I will remove it.